lib/cypress/cqm_execution_calc.rb

```diff
@@ -36,14 +36,13 @@ def request_for(measure)
       end
 
       results = JSON.parse(response)
-
       # TODO: Change the return format of this to still include the measure_id and record_id
       # since they are necessary for ease of use in some calculators
       results.map do |_, result|
         result.map do |_, pop_criteria|
           pop_criteria.slice!('IPP', 'DENOM', 'DENEX', 'NUMER', 'measure_id', 'patient_id')
           pop_criteria['measure_id'] = BSON::ObjectId.from_string(pop_criteria['measure_id'])
-          pop_criteria['patient_id'] = BSON::ObjectId.from_string(pop_criteria['patient_id'])
+          pop_criteria['patient_id'] = @patients.select { |p| p.id == BSON::ObjectId.from_string(pop_criteria['patient_id']) }.first.tacomaPatient.id
```

Comment:
It makes a bit more sense to me here to just add `@cqm_patient_mapping = @patients.map {|patient| [patient.id, patient.tacomaPatient.id] }.to_h` to to initializer and then this line can just be  `pop_criteria['patient_id'] = @cqm_patient_mapping[BSON::ObjectId.from_string(pop_criteria['patient_id'])]`, which avoids doing a full nested loop to search every time.